mlibrary/search
# Migrating `Pride.Util.MultiSearch` and `Pride.Util.SearchSwitcher`.

# Overview
`Pride.Util.MultiSearch` and `Pride.Util.SearchSwitcher` are only ever used in Search, and also do not import any logic from Pride. The `MultiSearch` and `SearchSwitcher` classes have been directly added and will be removed from Pride.

## Anything else?
`Pride.requestRecord` needed to be [added back to Pride](https://github.com/mlibrary/pride/pull/150) because of Prejudice's use, so the code has been reverted.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
